src/app/service-box/service-box.component.ts

```diff
@@ -9,9 +14,26 @@ import {
 	styleUrls: ['./service-box.component.scss']
 })
 export class ServiceBoxComponent implements OnInit {
-	public opened = false;
+	
+	user: Observable<firebase.User>;
+	items: AngularFireList<any[]>;
+	msgVal: string = '';
+
+	constructor(public afAuth: AngularFireAuth, public af: AngularFireDatabase) {
+		this.user = this.afAuth.authState;
+	}
 
-	constructor() { }
+	logout() {
+    	this.afAuth.auth.signOut();
+	}
+
+	signInWithTwitter() {
```

Comment:
Create a class for parsing the results and storing them for appropriate tasks in future. This way we can able to extract some important meta data like picture url and text to display on to the screen.